packages/api/src/documentation-info.ts

```diff
@@ -26,3 +31,8 @@ export interface DocumentationInfo extends DocumentationBaseInfo {
   description: string;
   category: string;
 }
+
+export interface GoToInfo {
+  kind: 'Pod' | 'Container' | 'Image' | 'Volume';
+  info: PodInfo | ContainerInfo | ImageInfo | VolumeInfo;
```

Comment:
what if you add the type directly there PodInfo & { type : 'Pod'} | Container info & { type: 'container' ...

Here it's possible to say it's a Pod but provide an ImageInfo